pkg/trace/info/stats_test.go

```diff
@@ -6,6 +6,7 @@
 package info
 
 import (
+	"github.com/DataDog/datadog-agent/pkg/trace/sampler"
```

Comment:
This import is in the wrong place. I think the editor normally sorts them out if you have a `goimports` plugin. This should go with the rest of the `datadog-agent` import group.

Reply:
Fixed